### ISSUE TITLE
Lint/RedundantSafeNavigation

### ISSUE THREAD

**Issue by joelvh** [rubocop-hq/rubocop#8888]
The `Lint/RedundantSafeNavigation` rule is a false positive when formatting `Time`. The recommendation causes an exception if `time&.to_s(:short)` is `nil` and is changed to `time.to_s(:short)`.

--------

## Expected behavior

I expected Rubocop to ignore `to_s`, and in particular with an argument such as `to_s(:short)`.

## Actual behavior

The actual behavior was to recommend removing safe navigation for `to_s(:short)`, which causes an `ArgumentError` exception if there's a `nil`.

## Steps to reproduce the problem

`time&.to_s(:short)`

## RuboCop version

```
0.93.0 (using Parser 2.7.2.0, rubocop-ast 0.7.1, running on ruby 2.6.3 x86_64-darwin19)
```

**Comment by bbatsov** [comment]
See https://github.com/rubocop-hq/rubocop/pull/8884

**Comment by marcandre** [comment]
Fixed and released. Thanks for opening the issue!
